MAUI/Radial-Menu/Place-And-Drag-RadialMenu.md

```diff
@@ -0,0 +1,273 @@
+
+* [`Position`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui__SfRadialMenu_DragBeginEventArgs_Position) - Gets the Start position of the RadialMenu
+
+* [`Handled`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragBeginEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragBeginEventArgs_Handled) - Gets and Sets the boolean value for enabling and disabling the dragging of RadialMenu.
+
+To hook the [`DragBegin`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.html#Syncfusion_Maui_SfRadialMenu_DragBegin) event, and to get the start position and restricts the dragging, follow the code example:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:RadialSample"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.RadialMenu;assembly=Syncfusion.Maui.RadialMenu"
+             x:Class="RadialSample.MainPage">
+    <radialMenu:SfRadialMenu x:Name="radialMenu"
+                             DragBegin="RadialMenu_DragBegin"
+                             IsDragEnabled = "True",
+                             CenterButtonText="&#xe710;",
+                             CenterButtonFontFamily="Maui Material Assets" ,
+                             CenterButtonRadius = "30",
+                             CenterButtonFontSize = "26",
+                             CenterButtonBorderThickness = "3">
+    </radialMenu:SfRadialMenu>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.RadialMenu;
+
+namespace RadialSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfRadialMenu radialMenu = new SfRadialMenu()
+            {
+                IsDragEnabled = true,
+                CenterButtonText="&#xe710;",
+                CenterButtonFontFamily="Maui Material Assets" ,
+                CenterButtonRadius = 30,
+                CenterButtonFontSize = 26,
+                CenterButtonBorderThickness = 3
+            };
+
+            radialMenu.DragBegin += RadialMenu_DragBegin;
+            this.Content = radialMenu;
+        }
+
+        private void RadialMenu_DragBegin(object sender, DragBeginEventArgs e)
+        {
+            e.Handled = true;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Drag End
+
+This event get triggered when dragging is ended in RadialMenu with [`DragEndEventArgs`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html).
+
+* [`OldValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_OldValue) - Gets the Start position of the RadialMenu
```

Comment:
* [`OldValue`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.SfRadialMenu.DragEndEventArgs.html#Syncfusion_Maui_SfRadialMenu_DragEndEventArgs_OldValue) - Gets the Start position of the RadialMenu.